README.md

```diff
@@ -0,0 +1,607 @@
+# Tariff Interpreter
+
+___
+This Interpreter is a reverence implementation for computing a JSON formatted bike-sharing tariff model.
+
+## Contents
+
+___
+
+* [Why this interpreter](#why-this-tariff-schema)
+* [General Interpreter Operation](#general-interpreter-operation)
+* [Tariff JSON Schemas](#tariff-json-schema)
+
+## Why this Tariff-Schema
+
+___
+There are two main reasons why the creation of a new pricing model has begun:
+
+* our current style for represent price tariffs are functions. On the one hand, this has the disadvantage that it leads
+  to inconsistencies between the tariffs and, on the other hand, it is difficult to create and read a tariff.
+* The old model was not able to provide a transparent breakdown of costs incurred similar to a receipt.
+
+These points were tried to improve specifically.
+
+## General Interpreter Operation
+
+___
+The Interpreter requires a [Tariff JSON Object](#tariff-json-schema) and a rental period.
+
+## Tariff JSON-Schema
+
+___
+Basically, there are three types of tariffs:
+
+* [SlotBasedTariff:](#slotbasedtariff)
+
+This type of tariff can represent tariff that depend only on the duration of a rental.
+
+* [DayBasedTariff:](#daybasedtariff)
+
+  These tariffs depend not only on the rental period, but also on the number of days started.
+* [TimeBasedTariff:](#timebasedtariff)
+
+  These tariffs depend not only on the rental period, but also on the time of day. It is important to cover a whole
+  week.
+
+To make it as simple as possible, all tariffs consist of a modular principle. This allows you to build up your tariff by
+adding slots and rates to your tariff
+
+### SlotBasedTariff
+
+___
+
+| Field Name      | Required |                Type                | Defines                                                                           |
+|-----------------|:--------:|:----------------------------------:|-----------------------------------------------------------------------------------|
```

Comment:
I'm not too sure if the tables are necessary - they feel a little too much and duplicate. Wouldn't it be enough to just explain the "unclear" stuff like goodwill and what's a billing interval? When looking at the JSON below, I can see that it's an array of stuff.

Reply:
Yes I see your point, but I think it is important to see which values are nessesary and which are optional. But maybe there is a shorter overfew enough

Reply:
Or we keep the verbosity and add sub pages so it's more structured